docs/started.rst

```diff
@@ -57,6 +57,9 @@ ReFrame's latest stable version is available through different channels:
 
      eb easybuild/easyconfigs/r/ReFrame/ReFrame-VERSION.eb -r
 
+.. note::
+
+    The above methods perform a bare installation of ReFrame, therefore, unittests and tutorial examples are not included.
```

Comment:
I don't think that's true for the Spack installation. For sure it has the tutorial, because it installs it from Github.